CRM/Contact/BAO/Relationship.php

```diff
@@ -43,6 +43,12 @@ class CRM_Contact_BAO_Relationship extends CRM_Contact_DAO_Relationship {
    */
   const NONE = 0, EDIT = 1, VIEW = 2;
 
+  /**
+   * The list of column headers
+   * @var array
+   */
+  static $_columnHeaders;
```

Comment:
Can we ditch the leading underscore on new vars - we have been removing these a bit

Reply:
ok